chainlink-ccv/v1/aggregator.proto

```diff
@@ -99,7 +91,8 @@ message MessageWithVerifierResult {
   bytes ccv_data = 4;
   int64 timestamp = 5;
```

Comment:
timestamp and sequence are aggregator provided right / trust model is only aggregator? For those maybe group them separately and have a clear doc/comment

Reply:
oh also I'm a little confused on MessageWithVerifierResult and VerifierResult - VerifierResult also has the message?